### PR TITLE
COMMERCE-12443

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12443

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase [hunk 1]
```diff
@@ -287,6 +287,207 @@ definition {
...
+		task ("And Buyer User logged in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And buyer user adds a product to the cart and the price is shown acordoly") {
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCheckout.testcase:443]: There is a typo here. For "acordoly" do you mean "accordingly"?

**Review by pietrobocale — CHANGES_REQUESTED** [review]
Review Completed, asking for some small changes, check the comments

Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**